draft-private-access-tokens.md

```diff
@@ -80,10 +80,79 @@ separation between the parties involved in the issuance and the redemption
 protocols, it is possible to eliminate this information concentration without
 any functional regressions. This document describes such a protocol.
 
+
 ## Requirements
 
 {::boilerplate bcp14}
 
+
+# Motivation
+
+This section describes classes of use cases where an origin would traditionally
+use a stable unique client identifier for enforcing policy. Hiding these
+identifiers from origins would therefore require an alternative for origins to
+continue enforcing their policies. Using the Privacy Address Token architecture
+for addressing these use cases is described in {{examples}}.
+
+## Rate-limited Access
+
+An origin provides rate-limited access to content to a client over a fixed
+period of time. The origin does not need to know the client's identity, but
+needs to know that a requesting client has not exceeded the maximum rate set by
+the origin.
+
+A specific example of this use case is a metered paywall, where an origin limits
+the number of page requests to each unique user over a period of time before the
+user is required to pay for access. The origin typically resets this state
+periodically, say, once per month. For example, an origin may serve ten (major
+content) requests in a month before a paywall is enacted.
+
+Origins routinely use client IP addresses for this purpose.
```

Comment:
```suggestion
Origins routinely use client IP addresses for this purpose. The per-client policy enforcement
possible with Private Access Tokens allows for rate-limiting on the level of a client device or
user account, instead of client IP address.
```

Reply:
I don't think you should add this text here. This belongs in {{examples}}, where the solution using PATs is described.

Reply:
Fair, that works